djangocms_versioning/admin.py

```diff
@@ -46,7 +80,12 @@ class VersionAdmin(admin.ModelAdmin):
         'label',
     )
     list_display_links = None
-    list_select_related = ('content',)
+
+    def get_queryset(self, request):
```

Comment:
If you do the integration test mentioned above then you'll have this bit covered as well I think (and `get_changelist`). But if you don't do the above as an integration test then you should make sure these methods are tested too.